src/layout/MainLayout/Header/HeaderContent/index.tsx

```diff
@@ -4,10 +4,9 @@ import { useContext } from 'react';
 import { Box, useMediaQuery, useTheme } from '@mui/material';
 
 // project import
-import toolbarItems from '../../../../toolbar-items';
+// import toolbarItems from '../../../../toolbar-items';
```

Comment:
Is this coming back at some point? If not, please remove. Otherwise, can we add //TODO or comment why it is commented out?